pwndbg/gdblib/disasm/mips.py

```diff
@@ -152,5 +171,25 @@ def _resolve_target(self, instruction: PwndbgInstruction, emu: Emulator | None,
 
         return super()._resolve_target(instruction, emu, call)
 
+    @override
+    def _parse_memory(
+        self,
+        instruction: PwndbgInstruction,
+        op: pwndbg.gdblib.disasm.arch.EnhancedOperand,
+        emu: Emulator,
+    ) -> int | None:
+        """
+        Parse the `MipsOpMem` Capstone object to determine the concrete memory address used.
+        """
+        target = op.mem.disp
+
+        if op.mem.base != 0:
+            base = self._read_register(instruction, op.mem.base, emu)
+            if base is None:
+                return None
+            target += base
+
+        return target
```

Comment:
`return base` above and `return op.mem.disp` here?

Reply:
Tho the current code is also fine

Reply:
On a second look - I think base is always valid (non-zero), so I changed this as the riscv _parse_memory to simplify them.